_posts/2024-02-20-announcing-trino-fest-2024.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Trino Fest goes to Boston in 2024"
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: false
+---
+
+After the resounding success of Trino Fest and Trino Summit in 2023, Commander
+Bun Bun has exciting news to share: we're taking our biggest events of the year
+back to being in-person. They'll be hybrid, to be more specific, so if you can't
+travel, don't fret, you'll still be able to watch and ask questions in chat.
+But if you can travel, you won't want to miss out! Everything you already know
+and love about Trino Fest is moving to the East Coast for the lovely Boston
+summer. The event is on the 13th of June in the Hyatt Regency Boston, where
+we'll have a full day of talks, time to network, and a happy hour at the end of
+the day. You may even get to meet Commander Bun Bun, who's ditching the hiking
+gear in favor of training for the Olympics. Sound exciting?
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-fest-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=Blog-1">
+        Register to attend!
+    </a>
+</div>
+
+<!--more-->
+
+## Joining us live
```

Comment:
Add an easy anchor here as well

Reply:
Maybe 

Join us live

Reply:
or maybe we need to clarify this is about joining for in person as well as registering for virtual